db/version_set.h

```diff
@@ -804,6 +806,7 @@ class Version {
   int TEST_refs() const { return refs_; }
 
   VersionStorageInfo* storage_info() { return &storage_info_; }
+  const VersionStorageInfo* storage_info() const { return &storage_info_; }
```

Comment:
It's part of the cleanup that switches from `Version*` to `const Version*` in `CompactionIterator`, which I think is a good thing to do since `CompactionIterator` is not supposed to modify the `Version`. (With a `const Version*` though, we need a `const` method to be able to retrieve the `VersionStorageInfo`.)

I think this is actually a fairly common pattern (see e.g. `std::vector<T>::front`).